core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/BootMonitors.scala

```diff
@@ -400,20 +410,40 @@ object BootMonitors extends LazyLogging {
     dataSource: SlickDataSource,
     samDAO: SamDAO,
     workspaceManagerDAO: WorkspaceManagerDAO,
+    billingProfileManagerDAO: BillingProfileManagerDAO,
     gcsDAO: GoogleServicesDAO
-  ) =
+  ) = {
+    val billingRepo = new BillingRepository(dataSource)
+    val monitorRecordDao = WorkspaceManagerResourceMonitorRecordDao(dataSource)
+
+    val bpmBillingProjectDeleteRunner = new BPMBillingProjectDeleteRunner(
+      samDAO,
+      gcsDAO,
+      workspaceManagerDAO,
+      billingRepo,
+      new BpmBillingProjectLifecycle(samDAO,
+                                     billingRepo,
+                                     billingProfileManagerDAO,
+                                     workspaceManagerDAO,
+                                     monitorRecordDao
+      )
+    )
+
     system.actorOf(
       WorkspaceResourceMonitor.props(
         config,
         dataSource,
         Map(
           JobType.AzureLandingZoneResult ->
-            new LandingZoneCreationStatusRunner(samDAO, workspaceManagerDAO, new BillingRepository(dataSource), gcsDAO),
+            new LandingZoneCreationStatusRunner(samDAO, workspaceManagerDAO, billingRepo, gcsDAO),
           JobType.CloneWorkspaceContainerResult ->
-            new CloneWorkspaceContainerRunner(samDAO, workspaceManagerDAO, dataSource, gcsDAO)
+            new CloneWorkspaceContainerRunner(samDAO, workspaceManagerDAO, dataSource, gcsDAO),
+          JobType.AzureBillingProjectDelete -> bpmBillingProjectDeleteRunner,
+          JobType.OtherBpmBillingProjectDelete -> bpmBillingProjectDeleteRunner
```

Comment:
Why is the mapping for `GoogleBillingProjectDeleteRunner` not here?

Reply:
I removed the `GoogleBillingProjectDeleteRunner`, because I put back the sync behavior for GCP projects, in order to avoid breaking swatomation

Reply:
But the class still exists. Should it then be deleted?

Reply:
I removed all usages, but not the class itself. Deleted.